pkg/query/api/v1.go

```diff
@@ -210,6 +210,7 @@ func (api *API) query(r *http.Request) (interface{}, []error, *apiError) {
 		warnmtx.Unlock()
 	}
 
+	enableDeduplication = true
```

Comment:
please move it above to var definition: 

 var (
  ...,
  enableDeduplication = true
)

Reply:
same all below - it will be bit simpler.